### ISSUE TITLE
Consider Converting Slider Block Library

### ISSUE THREAD

**Issue by eric-michel** [pressden/emma#129]
~~https://splidejs.com/ is in active development, and appears to have all the necessary features. Testing should be done first, then figure out a reasonable deprecation option for old block.~~

Make sure to reassess options between https://splidejs.com/ and https://swiperjs.com/. Swiper is more opinionated and larger, but seems to have a much broader user base.

This issue is ON HOLD until better block components are available for a block backend rebuild.

**Comment by eric-michel** [comment]
Also need to do research on a new slide paradigm. Consider a reusable-block-style in which reusable slides are created as a custom post type or other construct and are added to the slider block itself on the appropriate page.
